resources/views/users/index.blade.php

```diff
@@ -38,22 +35,32 @@
             </div>
         </div>
     </div>
-    @if(session()->has('success'))
-        <script>
-            window.addEventListener('DOMContentLoaded', function() {
-                var notification = document.getElementById('notification');
-                var notificationText = document.getElementById('notification-text');
-
-                var successMessage = "{{ session('success') }}";
-                if (successMessage) {
-                    notificationText.textContent = successMessage;
-                    notification.style.display = 'block';
+@stop
 
-                    setTimeout(function() {
-                        notification.style.display = 'none';
-                    }, 4000);
+@section('js')
+    <script>
+        $(document).ready(function () {
+            $('#table1').on('click', '.btn_delete', function () {
+                var id = $(this).data('id');
+                var url = "/admin/users/" + id;
+                var deleteConfirm = confirm("Are you sure to delete User with id : " + id + "?");
```

Comment:
I think the error message should be 
**Are you sure you want to delete this user?**